common/src/main/java/tachyon/Constants.java

```diff
@@ -145,6 +145,7 @@
   public static final String MASTER_WEB_PORT = "tachyon.master.web.port";
   public static final String MASTER_TEMPORARY_FOLDER = "tachyon.master.temporary.folder";
   public static final String MASTER_HEARTBEAT_INTERVAL_MS = "tachyon.master.heartbeat.interval.ms";
+  public static final String MASTER_TTL_INTERVAL_MS = "tachyon.master.ttl.interval.ms";
```

Comment:
Oh, after reading more code, it seems this property value is used to describe how often to kick off TTL checker, right? in that case, I think it is better called something like `tachyon.master.ttlchecker.interval.ms`. Otherwise I thought it means some default ttl value.